nexus/catalog/migrations/v42__add_null_end_time_index_cdc_batches.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS idx_cdc_batches_end_time_null ON peerdb_stats.cdc_batches(end_time) WHERE end_time IS NULL;
```

Comment:
```suggestion
CREATE INDEX IF NOT EXISTS idx_cdc_batches_end_time_null ON peerdb_stats.cdc_batches(flow_name) WHERE end_time IS NULL;
```

Reply:
I think we already have one for flow name, let me check

Reply:
Yes we do https://github.com/PeerDB-io/peerdb/blob/main/nexus/catalog/migrations/V9__mirror_stats_rels.sql#L23